src/email-only-signup.js

```diff
@@ -94,5 +102,12 @@ export default {
 
 			return options;
 		}
+
+		// @param {Boolean} show - true: display the component. false: hide it
+		function toggleVisibility(show) {
+			el.classList.toggle('o-email-only-signup__visually-hidden', !show);
```

Comment:
Second argument to `toggle` is unavailable on some older Android webkits (e.g. Samsung browser pre 4.3)